### PR TITLE
Update Benchmarks

### LINKED ISSUE #1360: `bounty_count` and `block_number` seem to be incorrect when benchmarking `pallet_bounties`
*Log position:*
1. Before [this line](https://github.com/darwinia-network/substrate/blob/darwinia-v0.12.3/frame/bounties/src/benchmarking.rs#L142)
```rust
log::error!("benchmark claim_bounty bounty_count {:?}, blocknumber: {:?}", BountyCount::<T>::get(),
			frame_system::Pallet::<T>::block_number());
```
2. Before [this line](https://github.com/darwinia-network/substrate/blob/darwinia-v0.12.3/frame/bounties/src/lib.rs#L554)
```rust
log::error!("claim_bounty: {:?}, {:?}", bounty_id, unlock_at);
```
### Benchmark `pallet_bounties - claim_bounty` in darwinia-common --chain pangolin-dev
```shell
# cmd
./target/release/drml benchmark --header .maintain/lincense-header --execution wasm --heap-pages 4096 --steps 1 --repeat 1 --chain pangolin-dev --output node/runtime/pangolin/src/weights/ --extrinsic 'claim_bounty' --pallet 'pallet_bounties'
# log
2022-07-28 18:48:26 `pallet_timestamp::UnixTime::now` is called at genesis, invalid value returned: 0
2022-07-28 18:48:26 [0] 💸 generated 1 npos voters, 1 from validators and 0 nominators
2022-07-28 18:48:26 [0] 💸 generated 1 npos voters, 1 from validators and 0 nominators
2022-07-28 18:48:26 [0] 💸 new validator set of size 1 has been processed for era 1
2022-07-28 18:48:27 benchmark claim_bounty bounty_count 0, blocknumber: 0
2022-07-28 18:48:27 claim_bounty: 0, 57600
2022-07-28 18:48:27 ====================================================

2022-07-28 18:48:27 benchmark claim_bounty bounty_count 0, blocknumber: 0
2022-07-28 18:48:27 claim_bounty: 0, 57600
2022-07-28 18:48:27 ====================================================

2022-07-28 18:48:27 benchmark claim_bounty bounty_count 1, blocknumber: 57600
2022-07-28 18:48:27 claim_bounty: 1, 115200
Error: Input("Premature")
```
### Benchmark `pallet_bounties - claim_bounty` in polkadot --chain kusama-dev
```shell
# cmd
./target/release/polkadot benchmark pallet --execution wasm --heap-pages 4096 --steps 1 --repeat 1 --chain kusama-dev --extrinsic 'claim_bounty' --pallet 'pallet_bounties'
# log
2022-07-28 20:40:21 [0] 💸 generated 1 npos voters, 1 from validators and 0 nominators
2022-07-28 20:40:21 Took active validators from set with wrong size
2022-07-28 20:40:21 Took active validators from set with wrong size
2022-07-28 20:40:21 Took active validators from set with wrong size.
2022-07-28 20:40:21 Took active validators from set with wrong size
2022-07-28 20:40:22 benchmark claim_bounty bounty_count 0, blocknumber: 0
2022-07-28 20:40:22 claim_bounty: 0, 57600
2022-07-28 20:40:22 ====================================================

2022-07-28 20:40:22 benchmark claim_bounty bounty_count 0, blocknumber: 0
2022-07-28 20:40:22 claim_bounty: 0, 57600
2022-07-28 20:40:22 ====================================================

2022-07-28 20:40:22 benchmark claim_bounty bounty_count 0, blocknumber: 0
2022-07-28 20:40:22 claim_bounty: 0, 57600
2022-07-28 20:40:22 ====================================================
```
### A temporary solution
Set block_number to 0 in the [first step of benchmark](https://github.com/darwinia-network/substrate/blob/darwinia-v0.12.3/frame/bounties/src/benchmarking.rs#L141) `claim_bounty` via 
```rust
frame_system::Pallet::<T>::set_block_number(T::BlockNumber::zero());
```

### PR DESCRIPTION
**Pull request by AurevoirXavier**
Close #1360, #1381.

Pick https://github.com/paritytech/substrate/pull/10470 to darwinia-v0.12.4.

### CODE REVIEW

**Comment by AurevoirXavier** [comment]
After this gets merged, we could release a new version.

File: node/runtime/pangoro/src/lib.rs [hunk 1]
```diff
@@ -693,11 +693,23 @@ sp_api::impl_runtime_apis! {
 			use frame_system_benchmarking::Pallet as SystemBench;
 			use baseline::Pallet as BaselineBench;
 
-
 			impl frame_system_benchmarking::Config for Runtime {}
 			impl baseline::Config for Runtime {}
 
-			let whitelist: Vec<TrackedStorageKey> = vec![];
+			let whitelist: Vec<TrackedStorageKey> = vec![
```
Comment (AsceticBear) [inline node/runtime/pangoro/src/lib.rs:699]: What does the `whitelist` mean? Or any docs about this?

Comment (AurevoirXavier) [reply node/runtime/pangoro/src/lib.rs:699]: They have a cache.
There is no need to count r/w for the global-dup storage.

**Review by HackFisher — APPROVED** [review]

File: node/runtime/pangolin/src/pallets/vesting.rs [hunk 2]
```diff
@@ -5,7 +5,7 @@ use sp_runtime::traits::ConvertInto;
 use crate::{weights::pallet_vesting::WeightInfo, *};
 
 frame_support::parameter_types! {
-	pub const MinVestedTransfer: Balance = 100 * MILLI;
+	pub const MinVestedTransfer: Balance = COIN;
```
Comment (AsceticBear) [inline node/runtime/pangolin/src/pallets/vesting.rs:8]: Is this expected? @HackFisher

Comment (AurevoirXavier) [reply node/runtime/pangolin/src/pallets/vesting.rs:8]: Transfer 0.1 vested COIN is meaningless.

Comment (AurevoirXavier) [reply node/runtime/pangolin/src/pallets/vesting.rs:8]: If the token's price is 10$, then I think 0.1 is reasonable.

Comment (AsceticBear) [reply node/runtime/pangolin/src/pallets/vesting.rs:8]: image

It seems this const is only used here for validation. It won't introduce other issues.

**Review by AsceticBear — APPROVED** [review]

File: node/runtime/pangolin/src/pallets/vesting.rs (hunk 2 above)
Comment (HackFisher) [reply node/runtime/pangolin/src/pallets/vesting.rs:8]: LGTM.

**Pull request merged**